lib/components/ResourceList/ResourceList.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { ResourceList } from './ResourceList';
+
+const items = [
+  {
+    title: 'Utlån av bøker',
+    description: 'Nasjonalbiblioteket',
+    avatar: {
+      name: 'Nasjonalbiblioteket',
+      imageUrl: 'https://altinncdn.no/orgs/nbib/nasjonalbiblioteket.png',
+      imageUrlAlt: 'nasjonalbiblioteket logo',
```

Comment:
A not super important question: Can some of this information perhaps be extracted from the other props?

Like the name and the imageUrlAlt? (Can image alt be standardized to name + "logo" for example? Since logo is invariant to language) That way we won't need a nested type to define resource, since the avatar prop could (maybe be renamed to logoUrl or something and) just be a straight forward string?

This is just a suggestion of course, and it might be a silly one 😅
I'm just thinking that since this is such a specialized and standardized component, we have the option of simplifying its interface quite a lot and that way make it even easier to use